modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/SleepFragment.kt

```diff
@@ -93,6 +96,16 @@ class SleepFragment : BaseDialogFragment() {
         }
         binding.buttonCancelTime.setOnClickListener { cancelTimer() }
         binding.buttonCancelEndOfEpisodeOrChapter.setOnClickListener { cancelTimer() }
+        binding.sleepTimeSettings.setOnClickListener {
+            close()
+            analyticsTracker.track(AnalyticsEvent.PLAYER_SLEEP_TIMER_SETTINGS_TAPPED)
+            val fragment = PlaybackSettingsFragment().apply {
+                arguments = Bundle().apply {
+                    putBoolean(SCROLL_TO_SLEEP_TIMER, true)
+                }
+            }
+            (activity as FragmentHostListener).showModal(fragment)
```

Comment:
I wonder if it would be nicer to do this the same as the BookmarksFragment and navigate to the settings section.
```
                        val fragmentHostListener = (activity as? FragmentHostListener)
                        fragmentHostListener?.apply {
                            closePlayer() // Closes player if open
                            openTab(R.id.navigation_profile)
                            addFragment(SettingsFragment())
                            addFragment(fragment)
                        }
```
At the moment, the page has black status and navigation bars.

<img  width="300" src="https://github.com/user-attachments/assets/674d6cae-a180-48eb-b58c-5e7333795b25" />

Reply:
I liked that. Check the change here: [a68464a](https://github.com/Automattic/pocket-casts-android/pull/2829/commits/a68464a0b990f2ca3d2a81a929e675c591659b7b)